pysatNASA/instruments/methods/cdaweb.py

```diff
@@ -488,6 +515,76 @@ def download(date_array, tag='', inst_id='', supported_tags=None,
     return
 
 
+def cdas_download(date_array, tag='', inst_id='', supported_tags=None,
+                  data_path=None):
+    """Download NASA CDAWeb CDF data using cdasws.
+
+    This routine is intended to be used by pysat instrument modules supporting
+    a particular NASA CDAWeb dataset.
+
+    Parameters
+    ----------
+    date_array : array-like
+        Array of datetimes to download data for. Provided by pysat.
+    tag : str
+        Data product tag (default='')
+    inst_id : str
+        Instrument ID (default='')
+    supported_tags : dict
+        dict of dicts. Keys are supported tag names for download. Value is
+        a dict with 'remote_dir', 'fname'. Inteded to be pre-set with
+        functools.partial then assigned to new instrument code.
+        (default=None)
+    data_path : str or NoneType
+        Path to data directory.  If None is specified, the value previously
+        set in Instrument.files.data_path is used.  (default=None)
+
+    Note
```

Comment:
@aburrell I put a note here referencing the naming convention for cdasws instrument names. You can find the names using cdasws, and I could describe that process.
Also to save a lot of duplicated notes I put this here instead of in the instruments. I could add a note in each instrument with the exact command to get the name for that dataset, but I'm not sure that's necessary.

Reply:
I think it's best to include the information here and avoid duplication.  Unfortunately, I still have no idea how I would find these names.  Maybe give an example on how to find one of them?